### PR TITLE
Drop EL6 support and clean up

### PR DESCRIPTION
**Pull request by evgeni**
- **drop EL6 support**
- **drop last bits of EL5 testing**
- **always test tracer**

### CODE REVIEW

File: images/Dockerfile.el7 [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM quay.io/centos/centos:7
 
+RUN sed -i -e 's/^mirrorlist/#mirrorlist/g; /^#baseurl/ s/^#//; /baseurl/ s/mirror/vault/g' /etc/yum.repos.d/CentOS-Base.repo
 RUN yum install -y epel-release https://yum.theforeman.org/client/nightly/el7/x86_64/foreman-client-release.rpm && yum clean all
-RUN yum install make python-setuptools python-tracer subscription-manager python-pip -y && yum clean all
+RUN yum install make python-setuptools python-tracer subscription-manager python-pip python2-mock python-unittest2 -y && yum clean all
```
Comment (chris1984) [inline images/Dockerfile.el7:5]: Looks like this PR got into this PR, was that intended? https://github.com/Katello/katello-host-tools/pull/160/files

**Review by chris1984 — COMMENTED** [review]
Left one comment but looks fine once that duplicate change is out of there.

File: test/test_katello/test_tracer/test_deb.py [hunk 2]
```diff
@@ -38,8 +38,9 @@ def test_needrestart_output(self, m):
         output_list = needrestart_output.split("\n")
         m.return_value = needrestart_output
         self.assertEqual(needrestart(), output_list)
-        self.assertEqual(m.call_args.args, (["needrestart", "-b"],))
-        self.assertEqual(m.call_args.kwargs, {"universal_newlines": True})
+        expected_args = (["needrestart", "-b"],)
```
Comment (chris1984) [inline test/test_katello/test_tracer/test_deb.py:41]: Same as above comment

**Comment by evgeni** [comment]
Yeah, it did contain #160 to see things still pass -- rebased now that 160 is merged.

**Review by ekohl — APPROVED** [review]
I can also rebase https://github.com/Katello/katello-host-tools/pull/155 after this.

**Pull request merged**